### PR TITLE
test `fluxctl release`

### PR DESCRIPTION
**Pull request by philwinder**

### CODE REVIEW

File: cmd/fluxctl/realease_command_test.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+			"service": string(flux.ServiceSpecAll),
+			"image":   "alpine:latest",
+			"kind":    string(flux.ReleaseKindExecute),
+		}},
+		{[]string{"--update-all-images", "--service=default/flux"}, map[string]string{
+			"service": "default/flux",
+			"image":   string(flux.ImageSpecLatest),
+			"kind":    string(flux.ReleaseKindExecute),
+		}},
+		{[]string{"--update-all-images", "--all", "--exclude=default/test,default/yeah"}, map[string]string{
```
Comment (squaremo) [inline cmd/fluxctl/realease_command_test.go]: Do we expect comma-separated values or multiple options like `--exclude foo --exclude bar`? (or both)

Comment (philwinder) [inline cmd/fluxctl/realease_command_test.go]: No multiple options (In fluxctl release at least). Tests were based upon current implementation.

File: cmd/fluxctl/check_release_cmd.go [hunk 2]
```diff
@@ -98,12 +98,14 @@ func (opts *serviceCheckReleaseOpts) RunE(cmd *cobra.Command, args []string) err
 		lastSucceeded = time.Now()
 	)
 
+	fmt.Println("for loop")
```
Comment (squaremo) [inline cmd/fluxctl/check_release_cmd.go]: Debugging remnant?

File: cmd/fluxctl/check_release_cmd.go [hunk 3]
```diff
@@ -98,12 +98,14 @@ func (opts *serviceCheckReleaseOpts) RunE(cmd *cobra.Command, args []string) err
...
 	for range time.Tick(time.Second) {
 		if retryCount > 0 {
 			fmt.Fprintf(w, "Last status (%s): %s\n", lastSucceeded.Format(time.Kitchen), prevStatus)
 			fmt.Fprintf(w, "Service unavailable. Retrying (#%d) ...\n", retryCount)
 		}
 
+		fmt.Println("GetRelease")
```
Comment (squaremo) [inline cmd/fluxctl/check_release_cmd.go]: Debugging remnant?

**Pull request merged**